source/our-stacks/back-end/index.md

```diff
@@ -1,6 +1,80 @@
----
-title: Back-end
-date: 2016-09-29 15:27:11
-layout: stack
----
-Total Back-end
\ No newline at end of file
+#Backend
+The separation of software systems into front and back ends simplifies development and separates maintenance. A rule of thumb is that the front (or "client") side is any component manipulated by the user. The server-side (or "back end") code resides on the server.
```

Comment:
I think that for the sake of consistency it should be `client-side (or "front end")`, so that it matches the `server-side (or "back end")` part :)